test/gggenome.jl

```diff
@@ -0,0 +1,16 @@
+
+@testset "GGGenome" begin
```

Comment:
you should also add a case for extractTopHit - so that function gets included in the tests as well

Reply:
Also you need to add your test file to tests/runtests.jl for it to be included in the test set